### PR TITLE
trakt_list collection by episode

### PR DESCRIPTION
**Pull request by wakemaster39**
previously the trakt_list was unable to produce a list of episodes for the lists *collection* or *watched*. This enables that ability. 

Also the trakt_emit plugin always emitted a *series_name* that was equivalent to the trakt_list with the *strip_dates* field enabled. Added this field to trakt_emit to sync the result of the two plugins by default.

### CODE REVIEW

File: flexget/plugins/input/trakt_list.py [hunk 1]
```diff
@@ -46,6 +46,16 @@
...
+    },
+    'collection_episodes': {
+        'title': lambda i: '%s (%s) S%02dE%02d' % (i['show']['title'], i['show']['year'], i['season'],
+                                                 i['episode']),
+        'series_name': lambda i: '%s (%s)' % (i['show']['title'], i['show']['year']),
+        'series_season': 'season',
+        'series_episode': 'episode',
+        'series_id': lambda i: 'S%02dE%02d' % (i['season'], i['episode']),
+        'trakt_show_id': 'show.ids.trakt',
+        'trakt_show_slug': 'show.ids.slug'
```
Comment (gazpachoking) [inline flexget/plugins/input/trakt_list.py]: Why is this needed instead of just the normal episode type?

**Comment by stevezau** [comment]
@wakemaster39 any updates on this?

**Comment by liiight** [comment]
I don't think this is relevant anymore, is it?
